pkg/webhook/mutation/pod/v2/metadata/containers.go

```diff
@@ -0,0 +1,32 @@
+package metadata
+
+import (
+	"context"
+
+	dtwebhook "github.com/Dynatrace/dynatrace-operator/pkg/webhook"
+	metacommon "github.com/Dynatrace/dynatrace-operator/pkg/webhook/mutation/pod/common/metadata"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func Mutate(context context.Context, metaClient client.Client, request *dtwebhook.MutationRequest) bool {
+	if !metacommon.IsEnabled(request.BaseRequest) {
+		return false
+	}
+
+	log.Info("adding metadata-enrichment to pod", "name", request.PodName())
+
+	// TODO: probably will only modify the iniContainer and pod, not the containers
+
+	return true
+}
+
+// TODO: probably not needed at all
+// func Reinvoke(request *dtwebhook.BaseRequest) bool {
```

Comment:
remove?

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/4576/commits/40829de385df0bd492cfbdc1b92568ba57cf7e1c